sys/fido2/ctap/ctap_mem.c

```diff
@@ -28,7 +28,7 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
-#ifdef BOARD_NATIVE
+#ifdef BOARD_NATIVE32
```

Comment:
```suggestion
#if defined BOARD_NATIVE || defined BOARD_NATIVE32
```

maybe? (same in other places)

Reply:
We also have `CPU_NATIVE` for that.